tests/parser/features/external_contracts/test_modifiable_external_contract_calls.py

```diff
@@ -51,20 +54,23 @@ def set_lucky(_lucky: int128) -> int128:
 """
 
     contract_2 = """
-class Bar():
-    def set_lucky(_lucky: int128) -> int128: pass
+class ModBar():
+    def set_lucky(_lucky: int128) -> int128: modifying
 
-modifiable_bar_contract: modifiable(Bar)
-static_bar_contract: static(Bar)
+class ConstBar():
+    def set_lucky(_lucky: int128) -> int128: constant
+
+modifiable_bar_contract: ModBar
+static_bar_contract: ConstBar
 
 @public
 def __init__(contract_address: address):
-    self.modifiable_bar_contract = contract_address
-    self.static_bar_contract = contract_address
+    self.modifiable_bar_contract.address = contract_address
+    self.static_bar_contract.address = contract_address
```

Comment:
Remove the `.address` syntax for now.
Then we can consider adding support for it later.